src/components/form/date-input/date-input.component.ts

```diff
@@ -31,6 +31,9 @@ export class DateInputComponent implements ControlValueAccessor, Validator, OnIn
   @Input()
   public formControl: FormControl;
 
+  @Input()
+  public isInValid: boolean;
```

Comment:
Can I be really annoying and point out this should be `isInvalid` without the capital `V`?

Reply:
Changed it to Invalid